src/Orchard.Cms.Web/Modules/Orchard.OpenId/JwtBearerConfigureOptions.cs

```diff
@@ -1,41 +0,0 @@
-﻿using Microsoft.AspNetCore.Builder;
```

Comment:
True. Anyway, it's not needed here as it's an authentication middleware, that can be configured from `Configure`, where the database settings are available (and already used, BTW).